modules/ROOT/pages/fd-deploy-app-from-rm.adoc

```diff
@@ -1,25 +1,30 @@
 = Deploy a Mule Application from Runtime Manager
 
-It might be the case that you are not allowed to deploy an application directly from Flow Designer. Your organization might require someone with the proper authority to deploy applications from Runtime Manager. You can share with that person a copy of your project as a `.jar` file to deploy your application with.
+Perhaps your organization requires someone with the proper authority to deploy Mule applications once the design phase has been completed.
+Flow Designer allows you to export the project as a `.jar` file and share it with those who are responsible for Mule app deployment.
+You'll need to use *Export mule application*, as it produces a `.jar` file that includes all the connector credentials and artifacts required by the Mule runtime.
 
-== Procedure
-
-. In Flow Designer, follow these steps:
-.. Ensure that your project is free of errors.
-.. Click the download icon, which is to the right of the Undo and Redo arrows in the top-left corner of the screen.
-.. Select *Download Package*.
+. In Flow Designer, ensure that your project is free of errors.
```

Comment:
This is a prerequisite to the deployment task, not the first step.

Consider identifying it as such, using a title, or working it into the intro text: "After designing a Mule application in Anypoint Flow Designer and ensuring that it is free of errors...."

Reply:
Fixed.